tests/test_dashboard_lower_dropdown_page.py

```diff
@@ -92,3 +87,60 @@ def test_manage_jenkins_dropdown_items_clickable_and_visible_09(self, locator):
         assert driver.is_clickable(locator)
         assert driver.is_visible(locator)
 
+    def test_goto_newItem_from_dropdown_10(self):
```

Comment:
Probably have to chance to follow naming convention

Reply:
that would be too long.